### PR TITLE
upgrade JDBC drivers

### PR DESCRIPTION
**Pull request by gavinking**

### CODE REVIEW

**Comment by gavinking** [comment]
Damn, hit https://hibernate.atlassian.net/browse/HHH-15889?filter=21172 again. We _really_ should report that one!

**Comment by gavinking** [comment]
@beikov I had to disable [one of your tests](https://hibernate.atlassian.net/browse/HHH-16317) on Oracle

**Comment by beikov** [comment]
I reported the driver issue to @loiclefevre from Oracle. He suggests to upgrade the driver to 21.9.0.0 for now.

File: settings.gradle [hunk 1]
```diff
@@ -183,25 +183,29 @@ dependencyResolutionManagement {
             version( "h2gis", h2gisVersion )
             version( "hsqldb", hsqldbVersion )
             version( "derby", derbyVersion )
-            version( "pgsql", "42.5.0" )
-            version( "mysql", "8.0.27" )
-            version( "oracle", "21.8.0.0" )
+            version( "pgsql", "42.6.0" )
+            version( "mysql", "8.0.33" )
+            version( "oracle", "23.2.0.0" )
+            version( "mariadb", "2.7.9" )
```
Comment (Sanne) [inline settings.gradle:189]: Any specific reason to not have opted for 3.1.4 (latest of mariadb) ?
 - https://mariadb.com/kb/en/about-mariadb-connector-j/

**Comment by Sanne** [comment]
> I reported the driver issue to @loiclefevre from Oracle. He suggests to upgrade the driver to 21.9.0.0 for now.

so.. should we do that? I wonder if we're not better off disabling that specific test on Oracle (it's broken after all, and apparently tracked appropriately), at least we'd be in a position to better detect other regressions, should they happen.

**Comment by beikov** [comment]
I'm ok with disabling the test for now and re-enabling it with the next driver update. Just note that the JDBC driver has a bug, the test is fine.

**Comment by beikov** [comment]
Created https://hibernate.atlassian.net/browse/HHH-16675 for this.

File: settings.gradle (hunk 1 above)
Comment (gavinking) [reply settings.gradle:189]: Yes because of https://hibernate.atlassian.net/browse/HHH-15889?filter=21172

**Review by jeandelavarene — COMMENTED** [review]
A TIMESTAMP column doesn't include any timezone and hence can't be converted to an OffsetDateTime. This is why the latest driver throws an exception. The previous versions of the driver was assuming a timezone (UTC) for the TIMESTAMP. This was wrong and corrected through a bug fix.

**Pull request merged**